i18n/bundle/bundle.go

```diff
@@ -78,12 +80,29 @@ func (b *Bundle) ParseTranslationFileBytes(filename string, buf []byte) error {
 }
 
 func parseTranslations(filename string, buf []byte) ([]translation.Translation, error) {
-	if buf == nil || len(buf) == 0 {
+	if len(buf) == 0 {
 		return []translation.Translation{}, nil
 	}
 
 	ext := filepath.Ext(filename)
 
+	// `github.com/pelletier/go-toml` has unwonted unmarshal functions,
```

Comment:
Also, I would pull this logic out into a separate `unmarshalTOML`.  Use it in `unmarshal` instead of in here.